frontend/src/__tests__/cypress/cypress/tests/mocked/hardwareProfiles/workbenchHardwareProfiles.cy.ts

```diff
@@ -30,6 +30,114 @@ import { mockDscStatus } from '~/__mocks__/mockDscStatus';
 import type { PodKind } from '~/k8sTypes';
 import { IdentifierResourceType, TolerationEffect, TolerationOperator } from '~/types';
 
+export const globalScopedHardwareProfiles = [
+  mockHardwareProfile({
+    name: 'small-profile',
+    displayName: 'Small Profile',
```

Comment:
These mocks are still duplicated between here and modelCustomizationForm.cy.ts. Can we factor them out into an exported value somewhere? I think this could be alongside the mockHardwareProfile function exported from __mocks__.